aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,58 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+
+- **grommet-theme-hpe v3.0.0** - This version ships with additional responsive breakpoints to help deliver the optimal user experience for HPE customers.
+  - Gain precise control over how page layouts respond and adapt.
+  - Implement grid layouts, such as a dashboard of cards, which gracefully adapt and flow.
+  - These additional breakpoints are breaking changes from v2 that will require some minor code changes to ensure backwards compatibility. A [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-2.x-to-3.x) is available to make this a smooth process.
+
+## March 9, 2022 - April 1, 2022
+
+### Communications
+
+- We are actively working on a process for receiving design contributions from members outside of the Design System core team.
+- We will begin this process by rolling out a way for designers to contribute small fixes and medium enhancements by leveraging the Figma Branching feature. Stay tuned!
+
+
+### Roadmap Updates
+
+- **Page Layouts**
+  - Updated the [Page Layouts Figma file](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2891%3A21273) to reflect new Page component in Grommet. Page component guidance and examples coming soon to the Design System site.
+- **Global Banner Notifications**
+  - Guidance and examples are now available on the [Design System site](/templates/global-banner-notifications).
+  - Theme-related styling is available on `grommet-theme-hpe v2.4.0`.
+- **Page Header**
+  - Finalized gathering use cases, annotating designs from service teams, and exploring anatomy. [Design ideations](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header) are in progress.
+  - Conducted a user study test on contextual navigation with regards to the "Ascending" Button. The goal was to understand user expectations when navigating up a navigation hierarchy and to determine the most intuitive name for this navigational component. Design decisions and guidance will be coming soon.
+- **Tabs**
+  - Work is in progress to provide design and development guidance for how Tabs should display when there is not ample screen width to fit all available tabs.
+  - Gathered use cases from stakeholders to understand scenarios when more than 5 tabs may be necessary on a single page.
+  - Design and best practice ideation is in progress.
+
+
+### Stay Current with the Latest Changes
+- Enhanced Grid documentation to help developers better understand [Grid's](/components/grid#grid-properties-and-key-concepts) capabilities and how to master this component to create dynamic layouts.
+- Added an example of a Form with a [single required FormField](https://design-system.hpe.design/templates/forms#single-required-formfield) on the site.
+- FileInput - Trash icon got replaced with a [Close icon](/components/fileinput). 
+- Wizard - Fixed template spacing issues between elements in [Figma](https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizard?node-id=0%3A1).
```

Comment:
What were the spacing issues? How does this impact the reader? Are their designs going to be affected? Broken? Depending on the answers to these questions, they should be addressed in this bullet point or removed.

Reply:
@britt6612 I don't think renaming the section would address my question/concern. Ultimately, I'd still like to understand the changes which have been made and know whether I should expect them to alter my designs. Or if they aren't going to affect anyone, then why worthy of mention?

Reply:
@vavalos5

Reply:
> @britt6612 I don't think renaming the section would address my question/concern. Ultimately, I'd still like to understand the changes which have been made and know whether I should expect them to alter my designs. Or if they aren't going to affect anyone, then why worthy of mention?

Designers are notified in Figma when a new change has been made to a component. They have the option to accept the changes or keep their current designs. As a designer, i'd like to know what changes have been made if I haven't come across the notification within Figma. These small tickets tend to be questions we often get within Office Hours. I think it's worth mentioning within this platform that a change has been made to XYZ if i'm trying to quickly find new changes made within Figma.

Reply:
what if we do a bullet point that states Figma changes then the sub-bullet points can be what you have here just an idea

Reply:
> As a designer, i'd like to know what changes have been made if I haven't come across the notification within Figma.

☝️  This is exactly what my comments are about and trying to get this bullet point to state what the changes were. As currently written, "fixed template spacing issues" is vague and I'd have to take extra steps to go investigate and do my own comparison. I think we deliver a better reader experience by providing more specificity and eliminate the extra work placed on the reader. 

To deliver the better experience we need to move beyond "a change has been made to XYZ" to "[this specific] change has been made to XYZ." In this case, what were the template spacing issues and how were they resolved? Or, depending on the change we may be able to get away without elaborating the spacing issues and simply "fixed spacing issues by [how the issue was resolved] (e.g. "by applying consistent spacing between buttons", "by applying desired spacing between the step number and step title" -- whatever the fix was).